packages/dx-react-scheduler/src/plugins/drag-drop-provider.jsx

```diff
@@ -162,35 +143,22 @@ export class DragDropProvider extends React.PureComponent {
       if (targetType === VERTICAL_TYPE) {
         this.timeTableRects = verticalTimeTableRects(
           draftAppointments, startViewDate, endViewDate,
-          excludedDays, viewCellsData, cellDuration, timeTableCells,
+          excludedDays, viewCellsData, cellDurationMinutes, timeTableCells,
         );
       } else {
         this.timeTableRects = horizontalTimeTableRects(
```

Comment:
timeTableRects and allDayRects are better to rename to reflect their meanings - draft appointment geometry information and metadata